lib/classes/Gallery.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Trendwerk\TrendPress;
+
+final class Gallery
+{
+    public function __construct()
+    {
+        add_action('init', [$this, 'replace']);
+    }
+
+    public function replace()
+    {
+        remove_shortcode('gallery');
+        add_shortcode('gallery', [$this, 'add']);
+    }
+
+    public function add($attributes)
+    {
+        if (empty($attributes['ids'])) {
```

Comment:
This doesn't seem to work on `[gallery]` without ID's, but normal galleries do. See [theme unit test data](https://wpcom-themes.svn.automattic.com/demo/theme-unit-test-data.xml).